target/linux/ramips/dts/mt7620a_tplink_re200-v1.dts

```diff
@@ -0,0 +1,158 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "tplink,re200-v1", "ralink,mt7620a-soc";
+	model = "TP-Link RE200 v1";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,57600n8";
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		
+		led_power: power {
+		    label = "re200-v1:green:power";
+		    gpios = <&gpio1 0 GPIO_ACTIVE_LOW>;
+		};
+		
+		lan {
+			label = "re200-v1:green:lan";
+			gpios = <&gpio2 0 GPIO_ACTIVE_LOW>;
+		};
+
+		wlan {
+			label = "re200-v1:green:wlan";
+			gpios = <&gpio1 3 GPIO_ACTIVE_LOW>;
+		};
+		
+		qss {
+		    label = "re200-v1:green:qss";
+		    gpios = <&gpio1 15 GPIO_ACTIVE_LOW>;
+		};
+		
+		led_2g1 {
+		    label = "re200-v1:red:2g";
+		    gpios = <&gpio0 2 GPIO_ACTIVE_LOW>;
+		};
+		
+		led_2g2 {
+		    label = "re200-v1:green:2g";
+		    gpios = <&gpio3 0 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio0 13 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		wps {
+			label = "wps";
+			gpios = <&gpio0 12 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+	};
+};
+
+
+&state_default {
+	gpio {
+	    ralink,group = "i2c", "uartf", "ephy", "wled", "nd_sd", "rgmii1", "spi refclk";
+		ralink,function = "gpio";
+	};
+};
+
+&spi0 {
+	status = "okay";
+
+	m25p80@0 {
```

Comment:
flash@0

Reply:
I copied that from another device, fixed.